### PR TITLE
translate qa-international-multilingual to zh-hans

### PR DESCRIPTION
**Pull request by cuihuan**
translate qa-international-multilingual to zh-hans
English title: International & multilingual web sites

### CODE REVIEW

File: questions/qa-international-multilingual.zh-hans.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  <h2 id="question"><a href="#question">问题</a></h2>
+    <p class="question">如何定义“国际化”或“多语言”网站？</p>
+    <p>“国际化”或“多语言”两个概念在部分场景下是可互换的，但是，当用来定义网站的时候，他们有非常明确的含义。清楚地区分这两个概念很重要。理解二者的差异，可以帮助您更准确的规划和定义网站需求。同时，二者在设计、开发和部署阶段都有非常明确的要求，这些要求通常需要不同的技术解决方案。
+    </p>
+</section>
+
+
+<section>
+  <h2 id="answer"><a href="#answer">答案</a></h2>
+    <p>简而言之，“国际化”是指网站面向全球用户，而“多语言”是指网站不止一种语言。国际化网站可以使用多种语言，也可以是单语言；同样的，多语言的网站可以面向全球用户，也可以仅面向本地用户。举个例子，W3C的站点是个国际化站点，因为他是为全球受众设计的。尽管该网站主要用英语编写，但有些页面包含其他语言，因此该站点也是个多语言网站。</p>
```
Comment (xfq) [inline questions/qa-international-multilingual.zh-hans.html]:
```suggestion
    <p>简而言之，“国际化”是指网站面向全球用户，而“多语言”是指网站不止一种语言。国际化网站可以使用多种语言，也可以是单语言；同样的，多语言的网站可以面向全球用户，也可以仅面向本地用户。举个例子，W3C的站点是个国际化站点，因为它是为全球受众设计的。尽管该网站主要用英语编写，但有些页面包含其他语言，因此该站点也是个多语言网站。</p>
```

File: questions/qa-international-multilingual.zh-hans.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    <p>多数公司的全球主页是原生支持“国际化”的。由于要面向全球用户，网站通常要求用户选择一个区域站点。此类区域站点往往使用单一语言的（通常是当地主要目标用户的通用语言）。选择区域后，会将用户重定向到同域名下的一个子站点或者当前区域的独立网站。无论哪种方式，良好的一致性设计，都会让网站的切换不影响用户体验，同时，保留国际站点的能力。</p>
```
Comment (xfq) [inline questions/qa-international-multilingual.zh-hans.html]:
```suggestion
    <p>多数公司的全球主页是原生支持“国际化”的。由于要面向全球用户，网站通常要求用户选择一个区域站点。此类区域站点往往使用单一语言的（通常是当地主要目标用户的通用语言）。选择区域后，会将用户重定向到同域名下的一个子站点或者当前区域的独立网站。无论采用何种方式，良好的一致性设计都会让用户觉得它仍然是同一个站点的一部分，并保留国际站点的感觉。</p>
```

「不影响用户体验」和「保留国际站点的**能力**」和原文意思不符，感觉有点过于意译了。

File: questions/qa-international-multilingual.zh-hans.html [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    <p>“多语言”网站可以存在多种形式。站点可以提供语言选择，选择后只呈现一种语言。同时，当站点的用户需要多语言或者站点内需要嵌入外文时，也可以在一个页面呈现多种语言。</p>
```
Comment (xfq) [inline questions/qa-international-multilingual.zh-hans.html]:
```suggestion
    <p>“多语言”网站可以存在多种形式。站点可以提供语言选择功能，选择后只呈现一种语言。同时，当站点的用户需要多语言或者站点内需要嵌入外文时，也可以在一个页面呈现多种语言，在线外语词典就是一种常见的例子。</p>
```

「语言选择功能」更容易理解一些。

另外，最后漏译了一句。

File: questions/qa-international-multilingual.zh-hans.html [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    <p>理解国际化和多语言站点的要求，将有助于我们决策和定义网站文件结构、硬件和软件的要求、内容的结构和标记、并为设计人员提供有价值的输入，以便在呈现中考虑可能的场景。网站的导航也取决于这些要求。通常国际化网站的导航，通过国家/地区和语言两个因素来引导用户，而多语言网站的导航可能只需要涉及语言选择。如果设计之初，没有考虑清楚到国际化和多语言的因素，事后的修订将是一个非常大的挑战。</p>
```
Comment (xfq) [inline questions/qa-international-multilingual.zh-hans.html]:
```suggestion
    <p>理解国际化和多语言站点的要求，将有助于我们决策和定义网站文件结构、硬件和软件的要求、内容的结构和标记，并为设计人员提供有价值的输入，以便在呈现中考虑可能的场景。网站的导航也取决于这些要求。通常国际化网站的导航，通过国家/地区和语言两个因素来引导用户，而多语言网站的导航可能只需要涉及语言选择。如果设计之初，没有考虑清楚到国际化和多语言的因素，事后的修订将是一个非常大的挑战。</p>
```

「决策和定义」的宾语到「内容的结构和标记」就结束了，可以把后面的顿号改为逗号。

File: questions/qa-international-multilingual.zh-hans.html [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+</section>
+
+
+<section>
+  <h2 id="bytheway"><a href="#bytheway">其他信息</a></h2>
+  <p>多语言网站不仅仅关注语言。在不同地区和文化上，信息的展示也有诸多差异。有些文化使用逗号作为千位分隔符和句点作为小数点，而有些文化恰好相反。比如，一个国家的1.547和另一个国家的1,547实际上可能是同一个数字。看似一个字符的差异，实际影响巨大。</p>
+  <p>日期和时间的呈现也是一个容易导致用户混淆的典型例子。当使用两位数字分别表示年、月和日时，实际表示的日期可能无法确认。比如，不同文化下可以表示为：DD/MM/YY、MM/DD/YY和YY/MM/DD。那么，格式为“01/02/03”的日期就可以解释为三个不同的日期</p>
```
Comment (xfq) [inline questions/qa-international-multilingual.zh-hans.html]:
```suggestion
  <p>日期和时间的呈现也是一个容易导致用户混淆的典型例子。当使用两位数字分别表示年、月和日时，实际表示的日期可能无法确认。比如，不同文化下可以表示为：DD/MM/YY、MM/DD/YY和YY/MM/DD。那么，格式为“01/02/03”的日期就可以解释为三个不同的日期。</p>
```

Comment (cuihuan) [reply questions/qa-international-multilingual.zh-hans.html]: 感谢，已经修复相关问题

**Comment by cuihuan** [comment]
@xfq @aphillips 
'then' in the last sentence in the answer section is 'than'?


> Revising any of these later is a significantly greater challenge then allowing for the possibility of international and multilingual aspects from the beginning.

**Comment by r12a** [comment]
>  'then' in the last sentence in the answer section is 'than'?

Thanks. I'll fix this when i process the translation for publication.

**Comment by r12a** [comment]
Btw, @cuihuan thanks for sending this translation, but just a reminder that it's usually best to check with me before starting.  If you do that, i'll update the template used by the english version of the article before you begin.   ;-)  That avoids me having to apply the changes for both the english and chinese files.

(Of course, the other reasons to check include checking whether we have someone already translating the article or whether we have plans to radically change (or occasionally even discontinue) the article.)

**Comment by cuihuan** [comment]
Ok, I'll do it next time ^_^ @r12a

**Comment by r12a** [comment]
I merged this, thanks.  Next step will be to retrofit the template updates.  @cuihuan if you'll be submitting the other 4 translations i just signed off within a short while, i'll wait until i have those translations before completing the publication process and announcements, since it's more efficient to do them all together. Let me know.

**Comment by cuihuan** [comment]
> I merged this, thanks. Next step will be to retrofit the template updates. @cuihuan if you'll be submitting the other 4 translations i just signed off within a short while, i'll wait until i have those translations before completing the publication process and announcements, since it's more efficient to do them all together. Let me know.

@r12a  Thank you for synchronizing my process. I will translate the other 4 articles next month with my classmates. About the end of May we can complete.  In my opinion , dealing with them together is efficient

**Pull request merged**